packages/mjml-section/src/index.js

```diff
@@ -299,35 +299,38 @@ export default class MjSection extends BodyComponent {
     }
 
     // this logic is different when using repeat or no-repeat
-    let [[vOriginX, vPosX], [vOriginY, vPosY]] = ['x', 'y'].map(coordinate => {
-      const isX = coordinate === 'x'
-      const bgRepeat = this.getAttribute('background-repeat') === 'repeat'
-      let pos = isX ? bgPosX : bgPosY
-      let origin = isX ? bgPosX : bgPosY
-
-      if (isPercentage(pos)) {
-        // Should be percentage at this point
-        const percentageValue = pos.match(/^(\d+(\.\d+)?)%$/)[1]
-        const decimal = parseInt(percentageValue, 10) / 100
-
-        if (bgRepeat) {
-          pos = decimal
-          origin = decimal
+    let [[vOriginX, vPosX], [vOriginY, vPosY]] = ['x', 'y'].map(
+      (coordinate) => {
+        const isX = coordinate === 'x'
+        const bgRepeat = this.getAttribute('background-repeat') === 'repeat'
+        let pos = isX ? bgPosX : bgPosY
+        let origin = isX ? bgPosX : bgPosY
+
+        if (isPercentage(pos)) {
+          // Should be percentage at this point
+          const percentageValue = pos.match(/^(\d+(\.\d+)?)%$/)[1]
+          const decimal = parseInt(percentageValue, 10) / 100
+
+          if (bgRepeat) {
+            pos = decimal
+            origin = decimal
+          } else {
+            pos = (-50 + decimal * 100) / 100
+            origin = (-50 + decimal * 100) / 100
+          }
+        } else if (bgRepeat) {
+          // top (y) or center (x)
+          origin = isX ? '0.5' : '0'
+          pos = isX ? '0.5' : '0'
         } else {
-          pos = (-50 + decimal * 100) / 100
```

Comment:
Mh i'm not sure about this file here ? it seems that the behavior changed ?

Reply:
@iRyusa yeah idk why the diff is weird, but it is only a indent change. its a lot more clear in a side-by-side view